tests/ga/test_monitor.py

```diff
@@ -428,7 +428,6 @@ def test_collect_and_send_events(self, mock_lib_dir, patch_send_event, patch_get
                          '<Param Name="ExtensionType" Value="" T="mt:wstr" />' \
                          '<Param Name="OSVersion" ' \
                          'Value="Linux:DISTRO_NAME-DISTRO_VERSION-DISTRO_CODE_NAME:platform-release" T="mt:wstr" />' \
-                         '<Param Name="GAVersion" Value="WALinuxAgent-{0}" T="mt:wstr" />' \
```

Comment:
Why wouldn't this be here? We are not removing this field, right?

Reply:
This particular test just tests the `collect_and_send_events` and the XML is created from the event fetched from the VM, along with the sysinfo added. The sysinfo does not populate any `GAVersion` now, which is why I removed it.